packages/core-web/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  'extends': [
+    '../../.eslintrc.js',
+    'plugin:vue/recommended',
```

Comment:
let's remove this, and the `vue/` rules

likely the browser environment needs to be enabled here, and the node environment disabled https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
Updated!

Reply:
`node: true` should be switched off as well (correct me if its not merged in the way I'm thinking),
let's put `env` as the first key per the order in the root `eslintrc` too